src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -784,8 +789,13 @@ private void joinConferenceRoom()
         }
     }
 
-    void setPresenceStatus(String statusMsg)
+    synchronized void setPresenceStatus(String statusMsg)
```

Comment:
I'm wondering if maybe `synchronized` should only cover the `if` condition and the state modification. I suppose that `jitsiMeetTools.setPresenceStatus(` will send all changes in order and with proper synchronization. This is in the context of knowing that sending XMPP update may block for a while and make even the NO-OP threads(`statusMsg.equals(jvbParticipantStatus)`) wait for it to finish.

This is what I mean:
```
synchronized(this) {
  if (statusMsg.equals(jvbParticipantStatus))
        {
            return;
        }
  jvbParticipantStatus = statusMsg;
}


if (mucRoom != null)
{
            // Send presence status update
            OperationSetJitsiMeetTools jitsiMeetTools
                = xmppProvider.getOperationSet(
                    OperationSetJitsiMeetTools.class);


            jitsiMeetTools.setPresenceStatus(mucRoom, statusMsg);
}
```

Reply:
Done.